src/langs/json/en.json

```
@@ -838,11 +837,28 @@
       "title": "Add-Ons",
       "pinned": "Pinned add-ons will appear here"
     },
+    "data": {
+      "title": "Data & Tags",
+      "empty": "Use tags or key/value data to organize documents",
+      "tags": "Tags",
+      "data": "Data"
+    },
     "projects": {
       "title": "Projects",
       "pinned": "Pinned projects will appear here"
+    },
+    "toc": {
+      "empty": "Sections organize your document with a table of contents",
+      "notes": "Notes",
+      "pageAbbrev": "p.",
+      "sections": "Sections",
+      "title": "Notes & Sections"
     }
   },
+  "notes": {
+    "empty": "No notes",
```

More instructional/actionable empty message:

```suggestion
    "empty": "Add notes to highlight and comment on your document",
```